system-int-test/test/cmr/system_int_test/search/collection_concept_retrieval_test.clj

```diff
@@ -164,6 +164,7 @@
                                                 :url-extension "html"})
             _ (is (= 200 (:status response)))]
         (is (search/mime-type-matches-response? response mt/html))
+        (is (= "C1200000016-PROV1" (:concept-id coll1) ))
```

Comment:
We should update this test to not rely on the autogenerated concept-ids (we can do that outside of this PR, and I see line 139 does the same thing). The way to fix this is to specify the concept-id as part of the ingest request on lines 118 and 119.

Reply:
Can you actually change this check to verify the concept-id is in the HTML response?
`(is (.contains ^String (:body response) (:concept-id coll1)))))`